content/rc/how-to/flush-db.md

```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
```

Comment:
Maybe it worth mentioning the word "replicas" as well- so the data is flushed over all replicas

Reply:
all of the replicas flush their data simultaneously